Resources/config/services.xml

```diff
@@ -43,6 +44,12 @@
             <tag name="payment.plugin" />
         </service>
 
+
+	    <service id="payment.form.valiton_payment_saferpay_type" class="%payment.form.valiton_payment_saferpay_type.class%">
+			<tag name="payment.method_form_type" />
+			<tag name="form.type" alias="saferpay" />
+        </service>
+
```

Comment:
Code style: please use 4 spaces for a tab and correct indentation. And please keep to the bundle namespaces (valiton_payment_saferpay.form ...)